bindings/nodejs/lib/types/models/api/IMilestoneUtxoChangesResponse.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import type { HexEncodedString } from '../../utils/hexEncodedTypes';
+/**
+ * Milestone.
+ */
+export interface IMilestoneUtxoChangesResponse {
+    /**
+     * The milestone index.
+     */
+    index: number;
+    /**
+     * The output IDs (transaction hash + output index) of the newly created outputs.
+     */
+    createdOutputs: HexEncodedString[];
```

Comment:
Don't we have an OutputId type?

Reply:
Mentioned it here: https://github.com/iotaledger/iota-sdk/issues/490

Reply:
We have one yea; `export type OutputId = string;`